src/ServiceControl.Transports.ASBS/AzureQuery.cs

```diff
@@ -286,6 +291,37 @@ public override async IAsyncEnumerable<IBrokerQueue> GetQueueNames(
         throw new Exception($"Could not find a ServiceBus named \"{serviceBusName}\"");
     }
 
+    async Task<HashSet<string>> GetValidNamespaceNames(CancellationToken cancellationToken = default)
```

Comment:
We do we need the `HashSet` at all if we're only ever going to return a single value?